app/src/test/java/de/westnordost/streetcomplete/quests/AddCyclewayTest.kt

```diff
@@ -65,16 +96,24 @@ class AddCyclewayTest {
     }
 
     @Test fun `applicable to road with nearby cycleway that is not aligned to the road`() {
-        val mapData = TestMapDataWithGeometry(listOf(
-            way(1L, listOf(1,2), mapOf(
-                "highway" to "primary",
-                "width" to "18"
-            )),
-            way(2L, listOf(3,4), mapOf(
-                "highway" to "cycleway"
-            ))
-        ))
-        val p1 = p(0.0,0.0)
+        val mapData = TestMapDataWithGeometry(
+            listOf(
+                way(
+                    1L, listOf(1, 2),
+                    mapOf(
+                        "highway" to "primary",
+                        "width" to "18"
+                    )
+                ),
+                way(
+                    2L, listOf(3, 4),
+                    mapOf(
+                        "highway" to "cycleway"
+                    )
+                )
+            )
+        )
+        val p1 = p(0.0, 0.0)
```

Comment:
This is a good example for why I think the style before was better = easier to read.

Maybe this is a matter of taste, but I got a problem with this deeper level on indentation and all those free floating `)` everywhere

Reply:
Again, the `indent` rule ([`9c2f170`](https://github.com/streetcomplete/StreetComplete/pull/3649/commits/9c2f170cfb7c138d01501e0c7ccbecc54f41b511#diff-33398a17f9b4bdf60a442b7ceee4338601d22794b2884515f80504bf63cb216c)).